gpytorch/variational/cholesky_variational_distribution.py

```diff
@@ -36,7 +35,7 @@ def __init__(self, num_inducing_points, batch_size=None):
 
     def initialize_variational_distribution(self, prior_dist):
         self.variational_mean.data.copy_(prior_dist.mean)
-        self.chol_variational_covar.data.copy_(torch.cholesky(prior_dist.covariance_matrix, upper=True))
+        self.chol_variational_covar.data.copy_(prior_dist.scale_tril)
```

Comment:
We probably want to use a different initialization for WhitenedVariationalStrategy though. We should probably be initializing the Cholesky of the inverse.

Reply:
Oh never mind I see what you did down below.